pkg/otlp/metrics/config.go

```diff
@@ -43,6 +48,16 @@ type translatorConfig struct {
 // TranslatorOption is a translator creation option.
 type TranslatorOption func(*translatorConfig) error
 
+// WithRemapping specifies that certain OTEL metrics (such as container.* and system.*) need to be
+// remapped to their Datadog counterparts because they will not be available otherwise. This happens
+// in situations when the translator is running as part of a Collector without the Datadog Agent.
```

Comment:
Using this option means that the pdata structs passed to the Translator cannot be used concurrently by other code since we modify them. Should we add a comment about this?

Reply:
> Now that I think about it, it does prepend the otel.* prefix to some metrics, so it changes existing metrics (not just add new ones); maybe that is worth mentioning indeed.

Yeah, I was more worried about this, but it's true that the runtime metrics already do some modifications. I think we should mention the name change.